frontend/src/components/forms/projectFormTabs/DockerFormTab.tsx

```diff
@@ -109,8 +109,8 @@ const DockerFormTab: FC<{ form: FormInstance }> = ({ form }) => {
   
   if (withTemplate) {
     switchClassName += ' template-switch-active'
-    scriptPlaceholder = "bash /shared/input/helloworld.sh > \"/shared/output/helloWorldTest\n"+
-    "bash /shared/input/helloug.sh > \"/shared/output/helloUGent\n"
+    scriptPlaceholder = "bash /shared/input/helloworld.sh > \"/shared/output/helloWorldTest\"\n"+
+    "bash /shared/input/helloug.sh > \"/shared/output/helloUGent\"\n"
```

Comment:
dit lijkt mij per ongeluk aangepast? Want dit heeft niet echt iets te maken met de cancel knop en de "No score" in de submission feedback

Reply:
Ah da was gwn nog een foutje in de placeholder vd template. Kdacht da rechtstreeks op frontend gepushed had ma blijkbaar zat ik nog op deze branch

Reply:
ok dan is alles goed, ik zal mergen